tirgei/Questioner-API-V2
# #163221709 Setup directory structure

#### What does this PR do?
This Pull Request creates the directory structure for version 2 of the API

#### Description of Task to be completed?
The tasks for this PR is:
- Create the modules
- Add a ```__init__.py``` file to each module
- Add the main run file
- Add the requirements file

#### How should this be manually tested?
Clone this repository on your machine:
```$ git clone https://github.com/tirgei/Questioner-API.git ```

CD into the project folder on your machine
```$ cd Questioner-API ```

Check out the branch for this PR
``` $ git checkout  ch-setup-directory-structure-163221709``


#### Pivotal Tracker story associated with this PR
[#163221709](https://www.pivotaltracker.com/story/show/163221709)

#### Screenshots
![directory2](https://user-images.githubusercontent.com/17829576/51183121-656f1e00-18e1-11e9-8eed-933ab3102a5c.png)

![directory2-tree](https://user-images.githubusercontent.com/17829576/51183126-67d17800-18e1-11e9-83ba-4dba896759be.png)
